src/libraries/System.Net.Http/src/System/Net/Http/Headers/HttpContentHeaders.cs

```diff
@@ -22,8 +22,7 @@ public ICollection<string> Allow
             {
                 if (_allow == null)
                 {
-                    _allow = new HttpHeaderValueCollection<string>(KnownHeaders.Allow.Descriptor,
-                        this, HeaderUtilities.TokenValidator);
+                    _allow = new HttpHeaderValueCollection<string>(KnownHeaders.Allow.Descriptor, this);
                 }
                 return _allow;
             }
```

Comment:
For future cleanup, this can be simplified to just:
```C#
public ICollection<string> Allow => _allow ??= new HttpHeaderValueCollection<string>(KnownHeaders.Allow.Descriptor, this);
```
Same for some other props below.